google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystem.java

```diff
@@ -134,6 +134,18 @@ public static CloudStorageFileSystem forBucket(String bucket) {
     return forBucket(bucket, userSpecifiedDefault);
   }
 
+  /**
+   * Creates new file system instance for {@code bucket}, with existing provider and configuration.
+   *
+   * @param bucketName the name of the bucket to initialize CloudStorageFileSystem Object.
```

Comment:
the name of the bucket -> name of the bucket

Reply:
Done